### PR TITLE
Update perbase to v0.2.1

### PR DESCRIPTION
**Pull request by sstadick**
Update the `perbase` tool to version 0.2.1 and remove noarch as it is linux only.

### CODE REVIEW

**Comment by sstadick** [comment]
@BiocondaBot please add label

**Comment by sstadick** [comment]
https://github.com/bioconda/bioconda-recipes/pull/24535 should be closed as I have updated the recipe substantially.

File: recipes/perbase/meta.yaml [hunk 1]
```diff
@@ -1,29 +1,42 @@
...
 build:
   number: 0
-  noarch: generic
 
 requirements:
   build:
-    - rust
+    - rust >=1.42
+    - clangdev
+    - {{ compiler('c') }}
```
Comment (nsoranzo) [inline recipes/perbase/meta.yaml:19]: That's a better solution to make the linter happy: https://bioconda.github.io/contributor/linting.html#skipping-persistently-on-a-per-recipe-basis

Comment (sstadick) [reply recipes/perbase/meta.yaml:19]: Thanks for the link! After fighting it for a while though, the compilers / what they bring with them are actually needed, so it ended up being needed.

Comment (nsoranzo) [reply recipes/perbase/meta.yaml:19]: OK, cool then!

**Comment by sstadick** [comment]
@nsoranzo are you able to review / approve this?

**Review by nsoranzo — APPROVED** [review]

**Comment by nsoranzo** [comment]
@BiocondaBot please merge

**Comment by sstadick** [comment]
@nsoranzo Thank you!

**Pull request merged**